apps/heft/src/utilities/TapableUtilities.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { Hook, FullTap } from 'tapable';
+
+export class TapableUtilities {
+  public static getTaps<THook extends Hook<unknown, unknown>>(hook: THook): FullTap[] {
+    const result: FullTap[] = [];
+
+    hook.intercept({
```

Comment:
Something to be aware of is that calling `.intercept()` automatically marks a `Hook` as used. I had to work around this in the timeline profiler PR by switching to a different check at call sites.